stag-library-compiler/src/main/java/com/vimeo/stag/processor/StagProcessor.java

```diff
@@ -104,27 +114,27 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
                 Set<Modifier> modifiers = variableElement.getModifiers();
                 if (modifiers.contains(Modifier.FINAL)) {
                     throw new RuntimeException("Unable to access field \"" +
-                                               variableElement.getSimpleName().toString() + "\" in class " +
-                                               variableElement.getEnclosingElement().asType() +
-                                               ", field must not be final.");
+                                                variableElement.getSimpleName().toString() + "\" in class " +
+                                                variableElement.getEnclosingElement().asType() +
+                                                ", field must not be final.");
                 } else if (modifiers.contains(Modifier.PRIVATE)) {
                     throw new RuntimeException("Unable to access field \"" +
-                                               variableElement.getSimpleName().toString() + "\" in class " +
-                                               variableElement.getEnclosingElement().asType() +
-                                               ", field must not be private.");
+                                                variableElement.getSimpleName().toString() + "\" in class " +
+                                                variableElement.getEnclosingElement().asType() +
+                                                ", field must not be private.");
                 }
 
                 Element enclosingClassElement = variableElement.getEnclosingElement();
                 TypeMirror enclosingClass = enclosingClassElement.asType();
 
-                if (!TypeUtils.isParameterizedType(enclosingClass) ||
-                    TypeUtils.isConcreteType(enclosingClass)) {
-                    mSupportedTypes.add(enclosingClass.toString());
+                if (TypeUtils.isParameterizedType(enclosingClass) ||
+                        TypeUtils.isConcreteType(enclosingClass)) {
```

Comment:
There is also new bug that's introduced with the support for instantiating these non-concrete parameterized types. In our network layer, we utilize abstract parameterized classes that are never instantiated, but only extended by other model objects. The current code tries to mistakenly instantiate these abstract model objects.

I should have made the `SuperAbstractDataList` class in the sample app `abstract` so that this situation would have come up in testing, but I made a mistake and forgot about the possibility.

If we can prevent a type adapter from being created for abstract classes (by checking the modifier before calling `mSupportedTypes.add` and `addToListMap` in this processor class) then the bug will be fixed.

So two things:
1. Can you make the `SuperAbstractDataList` class abstract
2. Can you add in checks to make sure type adapters are not generated for abstract classes